pyproject.toml

```diff
@@ -285,3 +285,10 @@ select = [
 "tools/linter/**" = [
     "LOG015" # please fix
 ]
+
+[variant.default-priorities]
+namespace = ["nvidia"]
+
+[variant.providers.nvidia]
+requires = ["nvidia-variant-provider>=0.0.1,<1.0.0"]
+plugin-api = "nvidia_variant_provider.plugin:NvidiaVariantPlugin"
```

Comment:
What is the state of this? In https://inspector.pypi.io/project/nvidia-variant-provider/0.0.0a0/packages/3f/cf/2ad80e216dfbfb53e6689b20bf504fc803bacdce596935d29647e70735f1/nvidia_variant_provider-0.0.0a0-py2.py3-none-any.whl/nvidia_variant_provider/__init__.py this looks like a stub.